app/models/spree/line_item_decorator.rb

```diff
@@ -0,0 +1,6 @@
+Spree::LineItem.class_eval do
```

Comment:
I went with class_eval to be consistent with all the rest. Or else we will have a single class using the prepend method. Unless there's great performance issue, can we stick with `class_eval` ?